crystlib.py

```diff
@@ -7,7 +7,7 @@
 # crystal details as parameters.
 
 
-class Crystal():
+class Crystal(dict):
```

Comment:
if this class does nothing more than a dictionary, simply use a dict:
```python
crystal = {
    'a': a,
    'b': b,
    ...
}

Reply:
Addressed in: 59fa0f7805e13bcd3c89c653d35783ae3cab01bb ea1b915ab97c9520c17c47a22ac785a9dfec6916